src/classes/analyzer/analyzer_util.py

```diff
@@ -0,0 +1,128 @@
+"""
+This file holds top level functions for interacting
+with the analyzer class structure.
+"""
+
+from git import Repo
+from pathlib import Path
+import logging
+from typing import Optional
+
+from src.classes.report.file_report import FileReport
+from src.classes.statistic import CodingLanguage
+from src.utils.project_discovery.project_discovery import ProjectFiles
+from src.classes.analyzer.base_file_analyzer import BaseFileAnalyzer
```

Comment:
why aren't we using the new import syntax here via the `__init__` file?

```suggestion
from src.classes.analyzer import BaseFileAnalyzer
```

Reply:
Disregard, leads to circular import